go/enclave/rpc/EstimateGas.go

```diff
@@ -86,14 +88,11 @@ func EstimateGasExecute(builder *CallBuilder[CallParamsWithBlock, hexutil.Uint64
 	// todo @siliev - add overhead when the base fee becomes dynamic.
 	publishingGas := big.NewInt(0).Div(l1Cost, batch.BaseFee)
 
-	// The one additional gas captures the modulo leftover in some edge cases
-	// where BaseFee is bigger than the l1cost.
-	publishingGas = big.NewInt(0).Add(publishingGas, gethcommon.Big1)
-
 	// Overestimate the publishing cost in case of spikes.
+	// given that we publish in a blob, the amount will be very low.
 	// Batch execution still deducts normally.
 	// TODO: Change to fixed time period quotes, rather than this.
-	publishingGas = publishingGas.Mul(publishingGas, gethcommon.Big2)
+	publishingGas = publishingGas.Mul(publishingGas, adjustPublishingGas)
```

Comment:
I'm not sure this is the fix we want. I'm thinking we must further play with the tx gas price to allow for it to bump up pending transactions rather than extending the margins here. It's okay for now